tests/python/VultureBearTest.py

```diff
@@ -42,85 +42,115 @@ def prep_file():
 
             return list(self.uut.run())
 
+    def verify_results(self, test_case, expected):
+        detected = dict((item.message, (item.affected_code[0].start.line,
+                                        item.affected_code[0].end.line,
+                                        item.confidence))
+                        for item in self.get_results(test_case))
+        self.assertEqual(detected, expected)
+
     def test_used_variable(self):
-        good_file = """
-        x = 2
-        print(x)
+        code = """\
+x = 2
+print(x)
         """
-        self.assertEqual(len(self.get_results(good_file)), 0)
+        self.verify_results(code, {})
 
     def test_unused_variable(self):
-        bad_file = """
-        b = 10
-        a = 12
-        print(a)
-        """
-        self.assertEqual(len(self.get_results(bad_file)), 1)
-
-    def test_unused_parameter(self):
-        bad_file = """
-        def test(a):
-            return 1
-
-        test(1)
+        code = """\
+b = 10
+a = 12
+print(a)
         """
-        self.assertEqual(len(self.get_results(bad_file)), 1)
+        self.verify_results(code, {"unused variable 'b'": (1, 1, 60)})
 
-    def test_unused_function(self):
-        bad_file = """
-        def test(a):
-            return a
-        """
-        self.assertEqual(len(self.get_results(bad_file)), 1)
+    def test_unused_arg(self):
+        code = """\
+def test(a):
+    return 1
 
-    def test_import_confidence(self):
-        bad_file = """
-        from os import *
-        import subprocess
+test(1)
         """
-        result = self.get_results(bad_file)
-        self.assertEqual(len(result), 1)
-        self.assertEqual(result[0].confidence, 95)
-
-    def test_class_confidence(self):
-        bad_file = """
-        class Name:
-
-            def __init__(self, name):
-                self.name = name
-                self.sur_name = 'Something'
+        self.verify_results(code, {"unused variable 'a'": (1, 1, 100)})
 
+    def test_import(self):
+        code = """\
+from os import *
+import subprocess
         """
-        result = self.get_results(bad_file)
-        self.assertEqual(len(result), 2)
-        self.assertEqual(result[0].confidence, 70)
-
-    def test_func_confidence(self):
-        bad_file = """
-        def hello(name):
-            print('Hello World')
+        self.verify_results(code, {
+            "unused import 'subprocess'": (2, 2, 90)
+        })
+
+    def test_class(self):
+        code = """\
+class Name:
+    def __init__(self, name):
+        self.name = name
+        self.surname = 'Something'
```

Comment:
> This change should probably be made in a separate PR though, right?

Hm alright^^ However you could just make a commit on top if you like inside this PR. Using test files is not a time-consuming change :) See for example https://github.com/coala/coala-bears/commit/b49553a8d62cae9989d0778728adb09fecb7fffb#diff-c53fab5561bd6a2c764cf31f7aae42b2R12

Anyway what I've just seen: Some old test cases are renamed and the code is reindented. If possible keep the old tests as intact as possible. This eases review. Everything else would belong to a test-refactor which you would like to avoid in this PR and should be also put inside a separate commit/PR :+1: If the new (not-existing) indentation is now required in the code samples, we should already introduce test-files^^